components/camel-azure/camel-azure-servicebus/src/main/java/org/apache/camel/component/azure/servicebus/ServiceBusConsumer.java

```diff
@@ -228,8 +233,23 @@ public void onFailure(Exchange exchange) {
             if (cause != null) {
                 getExceptionHandler().handleException("Error during processing exchange.", exchange, cause);
             }
+
             if (!getConfiguration().isDisableAutoComplete()) {
-                clientWrapper.abandon(message).subscribeOn(Schedulers.boundedElastic()).subscribe();
+                if (getConfiguration().isEnableDeadLettering() && (Objects.isNull(getConfiguration().getSubQueue()) ||
```

Comment:
You can use also ObjectHelper from camel-utils to be aligned with the codebase

Reply:
ok thanks I'll check.
First time here so happy discover new things

Reply:
so something like
`(ObjectHelper.isEmpty(getConfiguration().getSubQueue()) ||ObjectHelper.equal(getConfiguration().getSubQueue(), SubQueue.NONE)` ?

Reply:
Yes, something like that.